testing/s3mock/src/main/java/org/projectnessie/s3mock/S3Bucket.java

```diff
@@ -47,11 +50,23 @@ public Lister lister() {
     return (String prefix) -> Stream.empty();
   }
 
+  @Value.Default
+  public PutObject putObject() {
```

Comment:
Maybe rename to `storer` or `setter` to align with `lister` and `deleter`.

Reply:
done